content/en/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -42,6 +41,36 @@ spec:
     - containerPort: 80
 ```
 
+Below are three types of commonly used name constraints for resources.
+
+### DNS Subdomain Names
```

Comment:
@tengqm , Good addition. Some minor comments:
- Should Line 27 remain or remove as this new section is identical content?
- Is it easier to understand:
`Contain no more than 253 characters` versus `contain at most 253 characters`
- Should the first word in the list be capitalized?
- Line 59, `Most Kubernetes resource [type] names are required to conform to the DNS label ... `

Reply:
- removed line 27
- first item revised as suggested
- IMHO, first word is NOT supposed to be capitalized because it is not a complete sentence
- line 59 revised